scalahost/nsc/src/main/scala/scala/meta/internal/semantic/AttributesOps.scala

```diff
@@ -325,12 +328,16 @@ trait AttributesOps { self: DatabaseOps =>
                   val morePrecisePos = fun.pos.withStart(fun.pos.end).toMeta
                   val syntax = "[" + targs.mkString(", ") + "]"
                   success(morePrecisePos, syntax)
+                case g.Apply(select @ g.Select(qual, nme), _) if isSyntheticName(select) =>
+                  val pos = qual.pos.withStart(qual.pos.end).toMeta
+                  success(pos, s".${nme.decoded}")
```

Comment:
I thought that was the way to get the name of a symbol, is there a better method?